pom.xml

```diff
@@ -9,7 +9,7 @@
   <name>Reproducible Build Maven Plugin</name>
   <description>This Plug-In allows to generate byte-for-byte reproducible artifacts</description>
   <url>http://zlika.github.io/reproducible-build-maven-plugin</url>
-  
+
```

Comment:
Avoid committing files that only differ by whitespaces, that pollutes the diff.

Reply:
@Zlika sorry about that, I was not sure if this came from previous PR. I'll be more careful in the future :-)